packages/opticss/src/parseSelector.ts

```diff
@@ -66,7 +66,7 @@ export class CompoundSelector extends CombinedSelector<CompoundSelector> {
    * Set pseudo element type on this `CompoundSelector`.
    * @param The `selectorParser.Pseudo` to assign.
    */
-  setPseudoelement(pseudo: postcssSelectorParser.Pseudo) {
+  setPseudoelement(pseudo: postcssSelectorParser.Pseudo | undefined) {
```

Comment:
The only place this method is used is down below. To hit this code path (and keep the method), it needs to be a little more flexible. Undefined is already a valid value for `this.pseudoelement`.